gbfs-validator/gbfs.js

```diff
@@ -436,7 +444,32 @@ class GBFS {
     const vehicleTypesFile = t.find(a => a.type === 'vehicle_types')
     const freeBikeStatusFile = t.find(a => a.type === 'free_bike_status')
     const stationInformationFile = t.find(a => a.type === 'station_information')
-    const stationPricingPlans = t.find(a => a.type === 'system_pricing_plans')
+    const systemPricingPlans = t.find(a => a.type === 'system_pricing_plans')
+    const systemInformation = t.find(a => a.type === 'system_information')
+
+    const manifestUrl = systemInformation?.body?.[0]?.body?.data?.manifest_url
+
+    if (manifestUrl) {
+      try {
+        const body = await got.get(manifestUrl, this.gotOptions).json()
+
+        t.push({
+          body,
+          required: true,
+          type: 'manifest'
+        })
+      } catch (error) {
+        t.push({
+          url: manifestUrl,
+          recommanded: true,
```

Comment:
Fix the typo here:
```suggestion
          recommended: true,
```

Reply:
I've fixed that old typo in [b1c77e0](https://github.com/MobilityData/gbfs-validator/pull/104/commits/b1c77e0dff3f42ce018980a8a7b163e90cc525b1)